recipes/libaesgm/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "2013.1.1":
+    url: "https://github.com/xmake-mirror/libaesgm/archive/refs/tags/2013.1.1.tar.gz"
```

Comment:
The original page should be http://gladman.plushost.co.uk/oldsite/AES/index.php but it no longer exists. Both Fedora and other Linux distros are using an internal copy of their source files. So using this mirror look fine.

Extra info: https://packages.altlinux.org/en/p9/srpms/libaesgm/

Reply:
@uilianries 
Thanks a lot for your detail information!